readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Viktor535x.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Viktor535x.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add these links to the description
https://www.loom.com/share/da672f3d083c4247960d02d43f7d527a